tools/nxstyle.c

```diff
@@ -171,20 +238,21 @@ int main(int argc, char **argv, char **envp)
     }
   else
     {
-      fprintf(stderr, "Invalid number of arguments\n");
+      FATAL("Invalid number of arguments\n");
       show_usage(argv[0], 1);
     }
 
   instream = fopen(filename, "r");
   if (!instream)
     {
-      fprintf(stderr, "Failed to open %s\n", argv[1]);
+      FATALFL("Failed to open", argv[1]);
       return 1;
     }
 
   /* Are we parsing a header file? */
 
   hdrfile = false;
+  g_file_type = unknown;
   ext     = strrchr(filename, '.');
```

Comment:
Something wierd with the vertical alignment.  Either go with vertically aligned = signs, or remove spaces and go with the ragged look.  Either is fine, but not both.

Reply:
Unresolved.  Please make alignment style consistent.